src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -189,8 +264,30 @@ const AcademiaSignUpForm = React.memo(({ setSuccess, success, ...props }) => {
                     choices={institutionTypes}
                     onChange={e => setInstitutionType(e)}
                 />
+                <StyledSelect
+                    narrow
```

Comment:
Also `required`? Seems like everything should be required that is new but the course syllabus

Reply:
Our current select doesn't support required, nor the leafy green ui.I raised a request to add required to leafy green select. For now I'm just erroring out for now, if the mandatory fields are not filled.

Reply:
Got it, sounds good here!

Reply:
Unfortunately our select is built with a custom div and no html selects are used. It's not very easy to make the default fill out component, which we get for input/select tags out of box. https://jira.mongodb.org/browse/PD-1579- In case you are interested.